### PR TITLE
docs(): Add section for lazy loading routes with SystemJs

### PR DESCRIPTION
**Pull request by antonyRoberts**

### CODE REVIEW

**Comment by brandonroberts** [comment]
Will you move this under the top section where the Webpack example is and just make the same example with the blog but just use SystemJS instead. And remove the plunker link.

**Comment by antonyRoberts** [comment]
Made the changes.

File: docs/overview/code-splitting.md [hunk 1]
```diff
@@ -49,6 +49,33 @@ export const blogRoutes: Routes = [
 ];
 ```
 
+If you're using SystemJs instead of webPack use System.import, to lazy load child routes using the following
+
+In routes.ts
+```
+import { Routes } from '@ngrx/router';
```
Comment (brandonroberts) [inline docs/overview/code-splitting.md]: Clean up the spacing and lines:

```ts
import { Routes } from '@ngrx/router';

export const routes: Routes = [
  {
    path: '/',
    component: HomePageComponent
  },
  {
    path: '/blog', 
    loadComponent: () => System.import('src/blog-main')
        .then(module => module.BlogMainComponent),
    index:{ 
      loadComponent: () => System.import('src/blog-index')
         .then(module => module.BlogIndexComponent)
    },
    children: [
        { 
          path: '/post',
          loadComponent: () => System.import('src/blog-post')
            .then(module => module.BlogPostComponent)
        }
      ]
  }
```

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**